Committees/Firearms_Education_Director.adoc

```diff
@@ -0,0 +1,29 @@
+// Title of leadership position goes here
+= Firearms Education Director (CFED)
+:toc:
+
+== Requirements
+// List any requirements for leading the committee or working group
+* Sufficient training to conduct at least the Intro to Firearms course, with a plan for more advanced training
+* Completion of PLTCOM Introduction to Scientific Socialism
```

Comment:
```suggestion
```

Reply:
We should just stipulate that upon the completion of the course that it goes into effect.